src/model/status.nit

```diff
@@ -103,6 +103,9 @@ class MissionStatus
 	# Unlocked stars
 	var stars = new Array[MissionStar]
 
+	# The current best score for a given star (unlocked or not)
+	var star_score = new HashMap[MissionStar, Int]
```

Comment:
Sadly this will not be enough for the moment.

Nit serialization deserialize the HashMaps in a funny way that is rather "not cool" to use from javascript. Let's say two arrays, one with the keys and the other with the values... And since the API returns the raw deserialized version of the object, well... you see the point.

In a perfect world I would not be such a lazy person and I would have implemented the whole custom deserializer for the JSON API. One day, I will do it, eventually. I started eating brown bread so why not.

I do not have a perfect solution to propose but here one though about this.

The philosophy behind Missions is that for each Track, we have a TrackStatus specific for a Player. Same for each Mission that have a MissionStatus. Both contain a reference to the track/mission so the javascript can easily display the results. Why not do the same with StarStatus?

With this system, the JS can iterate over the stars status and display it simply, in the order they are given instead of looking up in the Map. It's a crappy solution but a solution.